### PR TITLE
Doc: releasenotes_1.6

### PR DESCRIPTION
**Pull request by fuzhongl**
Signed-off-by: fuzhongl <fuzhong.liu@intel.com>

### CODE REVIEW

**Comment by wenlingz** [comment]
@fuzhongl  please fix your code follow below:
New errors/warnings found, please fix them:
==============================================

/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/build/doc/rst/release_notes/release_notes_1.6.rst:153: WARNING: Bullet list ends without a blank line; unexpected unindent.
/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/build/doc/rst/release_notes/release_notes_1.6.rst:396: WARNING: Unknown target name: "tap".
/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/build/doc/rst/release_notes/release_notes_1.6.rst:411: WARNING: Unknown target name: "tap".

make[1]: Leaving directory '/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/doc'
make[1]: *** [Makefile:61: html] Error 1
make: *** [Makefile:159: doc] Error 2
Build step 'Execute shell' marked build as failure
Setting status of 438b07880da39599305ce317d21732af3e259ab1 to FAILURE with url  and message: 'Build finished. '
Using context: SDC/INDUSTRY/LOGICAL_PARTITION/HYBRID Build
Triggering a new build of RmAcrnHypervisor
PremergeMailAutoSend is disabled. Triggering skipped
Finished: FAILURE

**Comment by ClaudZhang1995** [comment]
OK to verify

File: doc/release_notes/release_notes_1.6.rst [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+
+Known Issues
+************
+- :acrn-issue:`4046` - [WHL][Function][WaaG] Error info popoup when run 3DMARK11 on Waag
+- :acrn-issue:`4047` - [WHL][Function][WaaG] passthru usb, Windows will hang when reboot it
+- :acrn-issue:`4313` - [WHL][VxWorks] Failed to ping when VxWorks passthru network
+- :acrn-issue:`4520` - efi-stub could get wrong bootloader name
+- :acrn-issue:`4557` - [WHL][Performance][WaaG] Failed to run 3D directX9 during Passmark9.0 performance test with 7212 gfx driver
+- :acrn-issue:`4558` - [WHL][Performance][WaaG] WaaG reboot automatically during run 3D directX12 with 7212 gfx driver
+- :acrn-issue:`4560` - HL][SIT][HV]build HV fail in docker
```
Comment (NanlinXie) [inline doc/release_notes/release_notes_1.6.rst:117]: [WHL]

**Comment by ClaudZhang1995** [comment]
OK to verify

**Comment by NanlinXie** [comment]
@deb-intel , Please help to check and merge this PR. Thanks!

File: doc/release_notes/release_notes_1.6.rst [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+instructions in the :ref:`rt_industry_setup`.
+
+Version 1.6 major features
+**************************
+
+What's New in v1.6
+==================
+* Graphics pass-through support
+
+  - The hypervisor and Service OS support pass-through graphics device to target DM-launched guest VM, based on GVT-d.
```
Comment (gvancuts) [inline doc/release_notes/release_notes_1.6.rst:35]:
```suggestion
  - The hypervisor and Service VM support pass-through graphics device to target DM-launched guest VM, based on GVT-d.
```

**Pull request merged**